app/assets/javascripts/graphs.coffee

```diff
@@ -0,0 +1,155 @@
+$ ->
```

Comment:
& can we make these graphs use our colors? they're all in `app/assets/stylesheets/colors.sass`